eap6-support/hawkular-wildfly-agent-feature-pack-eap6/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml

```diff
@@ -622,11 +623,11 @@
         <operation-dmr name="Shutdown" internal-name="shutdown" > <!-- shutdown and restart -->
```

Comment:
I don't know. You tell me :) My thinking for not setting the modifies=true on those is that this isn't modifying anything (in the sense there is no persistent changes being made). We are just starting/stopping processes, but nothing is "changing" (where "change" is, say, a modification to a property stored in standalone.xml, as one example).